django_oidc_user/tests/settings.py

```diff
@@ -7,4 +7,12 @@
     'django.contrib.contenttypes',
     'django.contrib.auth',
     'django_oidc_user',
+    'mozilla_django_oidc',
 ]
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'test',
```

Comment:
`NAME = ':memory:'` would put the database into memory, which should be fastest.